Tests/js/alloyeditor/buttons/assets/ez-alloyeditor-button-heading-tests.js

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
probably I have missed it on committing, however shouldn't these file to be added to .gitgnore as they are generated on `grunt test`?